config/accounts.go

```diff
@@ -255,11 +256,12 @@ func (bidderCodes *AlternateBidderCodes) IsValidBidderCode(bidder, alternateBidd
 		return false, fmt.Errorf(ErrAlternateBidderNotDefined, bidder, alternateBidder)
 	}
 
-	if len(adapterCfg.AllowedBidderCodes) == 0 {
+	if !adapterCfg.Enabled {
+		// config has bidder entry but is not enabled, report it
 		return false, fmt.Errorf("alternateBidderCodes disabled for %q, rejecting bids for %q", bidder, alternateBidder)
 	}
 
-	if adapterCfg.AllowedBidderCodes[0] == "*" {
+	if len(adapterCfg.AllowedBidderCodes) == 0 || adapterCfg.AllowedBidderCodes[0] == "*" {
```

Comment:
I feel it would be unnecessary flexibility to allow user to add bidder codes along with `*` in the array. The array should be either `*` or a list of bidder codes.

Let me know what you this, I'll update the code accordingly.

Reply:
After reading the requirements again, I realized that it specifically says that the wild card should be at index 0 so I think what you have is good. No change needed.